src/components/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import Image from "next/image";
+
+const Header = () => {
+  return (
+    <header className="border-b">
+      <div className="container mx-auto px-4 h-16 flex items-center justify-between">
+        {/* Logo on the left */}
+        <Link href="/" className="flex items-center">
+          <Image
+            className="rounded-md"
+            src="/MotherLogo.png"
+            alt="Logo"
```

Comment:
This alt is not ally friendly. Make it more descriptive:

"Mother Dao in nature" (just an example)